lib/logger/lib/logger.ex

```diff
@@ -420,6 +426,7 @@ defmodule Logger do
       the backend is added
 
   """
+  @spec add_backend(atom, Keyword.t) :: Supervisor.on_start_child
```

Comment:
This one I'm not really sure about: maybe returning `Supervisor.on_start_child/0` reveals too much about the internal details of how this is implemented?